### DISCUSSION TITLE
The TOP div tag don't full width on mobile

### DISCUSSION THREAD

**Discussion by IRediTOTO** [tailwindlabs/tailwindcss#4520] [Help] [answered]

**Comment by ghost** [comment, marked as answer]
Add some description. What you mean by:
> The TOP div tag

How help-request is this that you didn't spend a minute to describe your problem.

**Reply by IRediTOTO** [reply]
sorry my bad. I tried to delete this post but not work

**Reply by IRediTOTO** [reply]
Thank you for replie me
